### DISCUSSION TITLE
A look at the future of *quartz*

### DISCUSSION THREAD

**Discussion by vaelixd** [DoubleDotStudios/Quartz#34] [Announcements]
*quartz* has evolved majorly over less than a year. I have a plan moving forwards that I will share here. Any updates to this road map will be posted here. *quartz* is being worked on regularly and I can only do so much as one person.

### High priority
- Re-integrating macOS support
- Improved markdown rendering

### Medium Priority
- User specified fonts
- Improved theming
- .toml configuration support
- Printing support

### Low Priority
- Note exporting
- Mobile support

---

If you feel I should make certain things higher or lower priority please let me know.

### Other Notes
The second app in the *quartz* suite is now in development. *quartz* code is a code editor designed to fit into any desktop. It's planned to release with: 
- toml configuration support
- user specified fonts
- user specified color schemes
- keyboard shortcuts
- auto-formatting

I can't yet give an estimated release time since it's still in early development but updates will be posted in this discussion until the *quartz* code repository is public.

**Comment by Saqr-Sayed** [comment]
any notes app uses a tool bar. I think it is high priority. the live markdown rendering is a great idea too.
can I put feature suggestions here or add them in an issue as a feature request? or in the Ideas section?

**Reply by vaelixd** [reply]
You can open an issue with your idea and it will be marked as an enhancement.
